### DISCUSSION TITLE
apt-get update complains about

### DISCUSSION THREAD

**Discussion by pjz** [regolith-linux/regolith-desktop#694] [Q&A] [answered]
When I do `apt-get update` I get: 
```
W: Failed to fetch https://regolith-desktop.io/release-ubuntu-jammy-amd64/dists/jammy/InRelease  Certificate verification failed: The certificate is NOT trusted. The name in the certificate does not match the expected.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]
```

despite following the regolith-desktop.com instructions and installing the key from https://regolith-desktop.org/regolith.key .  I think it worked before; did the key change?

**Comment by kgilmer** [comment, marked as answer]
Hi @pjz , the key did not change but the TLS cert on `regolith-desktop.io` was broken (by me) and so we've migrated to `regolith-desktop.org`.  More details here: https://regolith-desktop.com/posts/repo-url-change-2.1/

**Reply by samip** [reply]
E: The repository 'https://regolith-desktop.org/release-ubuntu-jammy-amd64 testing Release' does not have a Release file.
